app/models/converted_money_presenter.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+# view object for converted money
+class ConvertedMoneyPresenter
+  def initialize(money, new_currency)
+    @old_amount = money
+    @new_currency = new_currency
+  end
+
+  def present
+    return not_converted if @old_amount.currency == @new_currency
+    converted
+  end
+
+  def converted
```

Comment:
@dissident instead making two method which returns hash with same structure I suggest to make one method which will call methods for each field which return values and this field methods this return values dependent to conditions.